cachito/workers/config.py

```diff
@@ -102,11 +141,17 @@ class Config(object):
     # other tasks aren't starved when processing a large archive.
     worker_prefetch_multiplier = 1
 
+    cachito_jaeger_exporter_endpoint: Optional[str] = ""
+    cachito_jaeger_exporter_port: Optional[int]
+    cachito_otlp_exporter_endpoint: Optional[str] = ""
```

Comment:
We try to keep cachito-specific configuration above the Celery configuration in the Config class (the options that don't start with cachito_ are Celery opts)